lib/FieldType/Mapper/FloatFormMapper.php

```diff
@@ -61,7 +83,42 @@ public function mapFieldDefinitionForm(FormInterface $fieldDefinitionForm, Field
     }
 
     /**
+     * @param FormInterface $fieldForm
+     * @param FieldData $data
+     */
+    public function mapFieldValueForm(FormInterface $fieldForm, FieldData $data)
+    {
+        $fieldDefinition = $data->fieldDefinition;
+        $formConfig = $fieldForm->getConfig();
+        $names = $fieldDefinition->getNames();
+        $label = $fieldDefinition->getName($formConfig->getOption('languageCode')) ?: reset($names);
+
+        $fieldForm
+            ->add(
+                $formConfig->getFormFactory()->createBuilder()
+                    ->create(
+                        'value',
+                        NumberType::class,
```

Comment:
@dpobel Keep in mind I'm not a frontend developer so I'm usually behind current state of HTML browser support. Fortunately it's can be easily fixed on our side.